config/routes.rb

```diff
@@ -355,12 +355,7 @@
       post '/confirmations' => 'personal_key#update'
     end
 
-    get '/verify/v2(/:step)' => 'verify#show', as: :idv_app
-    get '/verify/v2/password_confirm/forgot_password' => 'verify#show', as: :idv_app_forgot_password
-
     namespace :api do
-      post '/verify/v2/password_confirm' => 'verify/password_confirm#create'
-      post '/verify/v2/password_reset' => 'verify/password_reset#create'
       post '/verify/v2/document_capture' => 'verify/document_capture#create'
       delete '/verify/v2/document_capture_errors' => 'verify/document_capture_errors#delete'
```

Comment:
As a follow-up, I think we should consider removing the `/v2` from these paths to remove any traces of "FSMv2" naming conventions.

Reply:
Doing so directly wouldn't pose any issues, because all routes in rails are referred to by symbols right?

Reply:
Yeah I think it should hopefully be a matter of updating these strings and nothing else.